metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -1891,6 +1894,46 @@ class MetalsLanguageServer(
         .orNull
     }.asJava
 
+  @JsonRequest("metals/findTextInDependencyJars")
+  def findTextInDependencyJars(
+      params: FindTextInDependencyJarsRequest
+  ): CompletableFuture[util.List[Location]] = {
+    def readMask: Future[String] = Option(params.mask) match {
```

Comment:
Could we move most of the logic to FindTextInDependencyJars and just invoke it here with options?
```
findTextInJars.find(Option(params.mask), Option(params.content))
```